e2e/compilation-cases/package.json

```diff
@@ -11,6 +11,18 @@
     "@types/jquery": "3.5.29",
     "devextreme": "workspace:*",
     "jquery": "3.7.1",
-    "typescript": "4.9.5"
+    "typescript": "4.9.5",
+    "eslint": "catalog:",
+    "@eslint-stylistic/metadata": "catalog:",
+    "@eslint/eslintrc": "catalog:",
+    "@stylistic/eslint-plugin": "catalog:",
+    "@typescript-eslint/eslint-plugin": "catalog:",
+    "@typescript-eslint/experimental-utils": "5.62.0",
```

Comment:
Do we need this package? If so, can we use the @typescript-eslint/utils package instead? Since this package is already [deprecated](https://www.npmjs.com/package/@typescript-eslint/experimental-utils#note), it is not maintained.

Reply:
deleted it